Api/Client.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+    namespace Metrilo\Analytics\Api;
+
+    use \Metrilo\Analytics\Api\Validator;
+
+    class Client {
+    
+        const VERSION = '1.0.0';
+//        private $url  = 'https://postb.in/kzTiQWNl';
+        private $url  = 'https://postman-echo.com/post';
+    
+        public function __construct(
+            array $backendParams = []
+        ) {
+            $this->backendParams = $backendParams;
+            $this->validator     = new Validator();
+        }
+    
+        /**
+         * Create HTTP POST request to URL
+         *
+         * @param String $url
+         * @param Array $bodyArray
+         * @return void
+         */
+        public function post($url, $bodyArray = false)
+        {
+            $encodedBody = $bodyArray ? json_encode($bodyArray) : '';
+            $parsedUrl = parse_url($url);
+            $headers = [
+                'Content-Type: application/json',
+                'Accept: */*',
+                'User-Agent: HttpClient/1.0.0',
+                'Connection: Close',
+                'Host: '.$parsedUrl['host']
+            ];
+            return $this->curlCall($url, $headers, $encodedBody);
+        }
+    
+        /**
+         * CURL call
+         *
+         * @param string $url
+         * @param array $headers
+         * @param string $body
+         * @param string $method
+         * @return void
+         */
+        public function curlCall($url, $headers = [], $body = '', $method = "POST")
```

Comment:
I don't think that all of the request sending logic (`post` and `curlCall` methods) belongs here, it would be nicer if it lives in a separate class (maybe called `Connection`?)

Reply:
Done, additional class Connection have been added.